AdvLoggerPkg/Library/AdvancedLoggerLib/PeiCore/AdvancedLoggerLib.c

```diff
@@ -553,14 +553,9 @@ AdvancedLoggerGetLoggerInfo (
       // Memory Discovered Ppi.  At that time, the full in memory log buffer is allocated.
       //
 
-      if (FeaturePcdGet (PcdAdvancedLoggerPeiInRAM)) {
-        Pages =  FixedPcdGet32 (PcdAdvancedLoggerPages);
-        Type  =  EfiRuntimeServicesData;
-      } else {
-        Pages =  FixedPcdGet32 (PcdAdvancedLoggerPreMemPages);
-        // This is to avoid the interim buffer being allocated to consume 64KB on ARM64 platforms.
-        Type =  EfiBootServicesData;
-      }
+      Pages =  FixedPcdGet32 (PcdAdvancedLoggerPreMemPages);
+      // This is to avoid the interim buffer being allocated to consume 64KB on ARM64 platforms.
+      Type =  EfiBootServicesData;

```

Comment:
Now that the PCD is gone, you can get rid of the `Type` variable and just use EfiBootServicesData directly in the AllocatePages call.

Reply:
Good catch, fixed.